molgenis/molgenis-emx2
# feat (tailwind): init radio and radio group components

What are the main changes you did:
- Added base component for radio input
- Added component for creating a group of radio inputs
- Added e2e test and revised e2e docs

how to test:
- Navigate to story in the preview: https://preview-emx2-pr-4231.dev.molgenis.org/apps/tailwind-components/#/input/Radio.story
- Interact with all examples
- Run WAVE accessibility extension in the browser to ensure WCAG compliance

todo:
- [x] updated docs in case of new feature (see showcase app)
- [x] Add e2e test
- [x] add button that clears selected values
